Documentation/toolbox.md

```diff
@@ -44,7 +44,7 @@ spec:
         name: libmodules
       - name: mon-endpoint-volume
         mountPath: /etc/rook
-  hostNetwork: false
+  hostNetwork: true # if false, the "crd map" command will hang, see https://github.com/rook/rook/issues/2021
```

Comment:
Shouldn't `crd` be `rbd`?
Also could you move the comment on a new line before the `hostNetwork` line to align it with other yaml comments (see yamls in `cluster/` directories).

Reply:
Fixed typo and moved comment above.